index.js

```diff
@@ -137,7 +64,7 @@ xapp.init(
       console.warn("[FEATURE] Enabling schema stitching") // eslint-disable-line
       mergeSchemas().then(startApp)
```

Comment:
It looks like this won’t end up using the merged schema, no?

Reply:
I'm not entirely sure how merge shemas works internally, but looking closely [#master index.js](https://github.com/artsy/metaphysics/blob/master/index.js) nothing has quite changed as it doesn't accept any parameters. If `enableSchemaStitching` is `true` it should execute, and in the same way as it did before.

Reply:
Oh! actually i take that back. I see that this is chained from a promise. Will update.

Reply:
Aye, thanks 👍